### PR TITLE
fixed code style errors with flake8

### PR DESCRIPTION
**Pull request by amitsagtani97**
Added the tox.ini file for the configuration of flake8 and removed the coding style errors.

CI currently has issues, so the current status is as follow:

- [x] smallfile 
- [x] uperf
- [x] fio 
- [x] fs-drift
- [ ] hammerdb
- [x] pgbench
- [x] ycsb
- [x] iperf
- [x] sysbench

### CODE REVIEW

**Comment by aakarshg** [comment]
ok to test

**Comment by aakarshg** [comment]
> CI currently has issues, so the current status is as follow:
> 
> * [ ]  smallfile
> * [x]  uperf
> * [ ]  fio
> * [ ]  fs-drift
> * [ ]  hammerdb
> * [ ]  pgbench
> * [ ]  ycsb
> * [x]  iperf
> * [x]  sysbench

fio, smallfile, fs-drift seem to have issues getting the uuid, makes sense as it tries to do it after deleting the namespace... my hunch is its due to https://github.com/cloud-bulldozer/ripsaw/commit/f061f4e1abcb43c1ecfa127dce204f5980dac11d

ycsb, pgbench have the good ol' api version issue.. for which fixes are making through in ripsaw

hammerdb has the same api version issue, but also has a few other for which a fix is on its way in ripsaw.

**Comment by aakarshg** [comment]
> > CI currently has issues, so the current status is as follow:
> > 
> > * [ ]   smallfile
> > * [x]   uperf
> > * [ ]   fio
> > * [ ]   fs-drift
> > * [ ]   hammerdb
> > * [ ]   pgbench
> > * [ ]   ycsb
> > * [x]   iperf
> > * [x]   sysbench
> 
> fio, smallfile, fs-drift seem to have issues getting the uuid, makes sense as it tries to do it after deleting the namespace... my hunch is its due to [cloud-bulldozer/ripsaw@f061f4e](https://github.com/cloud-bulldozer/ripsaw/commit/f061f4e1abcb43c1ecfa127dce204f5980dac11d)
> 
> ycsb, pgbench have the good ol' api version issue.. for which fixes are making through in ripsaw
> 
> hammerdb has the same api version issue, but also has a few other for which a fix is on its way in ripsaw.

pgbench and ycsb have been fixed..

**Comment by aakarshg** [comment]
The only false positive seems to be from hammerdb, the test itself is not run as it fails on creation of the msql-server pod because of the API issue...

File: tox.ini [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+[testenv:pep8]
+basepython = python
+commands = flake8 {posargs}
+
+[flake8]
+show-source = True
+ignore = E123,E125,E226,E302,E41,E231,E203,H233,H306,H238,H236,H404,H405,W504
+max-line-length = 110
+builtins = _
+exclude=.venv,.git,.tox,*lib/python*
```
Comment (RobertKrawitz) [inline tox.ini]: Add a newline here

File: README.md [hunk 2]
```diff
@@ -214,3 +214,16 @@ index="ripsaw-your-wrapper-results"
 check_es $uuid $index
 exit $?
 ```
+
```
Comment (jtaleric) [inline README.md]: Maybe add 

```
## Style guide
Max line length is 110 to avoid linting issues.
```

**Comment by rsevilla87** [comment]
LGTM! nice work

**Comment by aakarshg** [comment]
> One small change, after executing `tox` on my local I had a .tox directory created. We should add it to .gitignore

Agreed, cc: @amitsagtani97

**Pull request merged**